tools/clang/test/HLSLFileCheck/validation/NonVLAArrays.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %dxc -E PSMain -T ps_6_0 %s | FileCheck %s
+
+// CHECK: define void @PSMain()
```

Comment:
We are determined to get away from tests that only check for an entry function. In this case, I think the most important part would be to verify that these cbuffers show up in the $Globals with the proper sizes.